llb/llb.proto

```diff
@@ -41,22 +51,27 @@ message Mount {
 	bool readonly = 5;
 }
 
+// CopyOp copies files across Ops.
 message CopyOp {
 	repeated CopySource src = 1;
 	string dest = 2;
 }
 
+// CopySource specifies a source for CopyOp.
 message CopySource {
 	int64 input = 1 [(gogoproto.customtype) = "InputIndex", (gogoproto.nullable) = false];
 	string selector = 2;
 }
 
+// SourceOp specifies a source such as build contexts and images.
 message SourceOp {
-	// source type?
+	// identifier e.g. local://, docker-image://, git://, https://...
```

Comment:
keep the comment as I still want to do it. Or make it more readable, "use source type or any type instead of URL protocol"